ingestion-edge/Dockerfile

```diff
@@ -1,21 +1,18 @@
 ARG PYTHON_VERSION=3.10
 
 # build requirements in separate stage because it requires gcc and libc-dev
-FROM python:${PYTHON_VERSION}-slim-buster AS base
+FROM python:${PYTHON_VERSION}-slim-bookworm AS base
```

Comment:
I'm wondering if there are any benefits to updating one version at a time but this is probably fine since there isn't that much happening here